src/moduler/aktivitet/ikke-delt-markering/IkkeDeltMarkering.tsx

```diff
@@ -3,6 +3,18 @@ import React from 'react';
 import EtikettBase from '../../../felles-komponenter/etikett-base/etikett-base';
 import styles from './ikke-delt-markering.module.less';
 import visibleIfHOC from "../../../hocs/visible-if";
+import {Aktivitet} from "../../../datatypes/aktivitetTypes";
+import {MOTE_TYPE, SAMTALEREFERAT_TYPE} from "../../../constant";
+import {useSelector} from "react-redux";
+import {selectErVeileder} from "../../identitet/identitet-selector";
+
+export function SkalIkkeDeltMarkeringVises({type, erReferatPublisert, referat}: Aktivitet): boolean {
```

Comment:
Skriv gjerne denne om til arrow function
```suggestion
export const SkalIkkeDeltMarkeringVises = ({type, erReferatPublisert, referat}: Aktivitet): boolean => {
```